code/api/src/modules/product/resolvers.js

```diff
@@ -37,8 +37,7 @@ export async function getRelated(parentValue, { productId }) {
     where: {
       id: { [models.Sequelize.Op.not]: productId }
     },
-    limit: 3,
-    order: [[models.Sequelize.fn('RAND')]] // mock related products by showing random products
```

Comment:
This call to randomize the order in which products were displayed was throwing an error when running tests. Removed this to address issue.

Reply:
Good fix, I was noticing that bug as well